arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -434,14 +434,87 @@ public async Task PostVertexAsync_ShouldReturnNewVertex_WhenReturnNewIsTrue()
                 Name = propertyName
             }, new PostVertexQuery
             {
-                 ReturnNew = true,
-                 WaitForSync = true
+                ReturnNew = true,
+                WaitForSync = true
             });
 
             Assert.Equal(HttpStatusCode.Created, response.Code);
             Assert.False(response.Error);
             Assert.NotNull(response.New);
             Assert.Equal(propertyName, response.New.Name);
         }
+
+        [Fact]
+        public async Task DeleteVertexCollectionAsync_ShouldSucceed()
+        {
+            // Create a new graph
+
+            string graphName = nameof(DeleteVertexCollectionAsync_ShouldSucceed);
+
+            PostGraphResponse createResponse = await _client.PostGraphAsync(
+                new PostGraphBody()
+                {
+                    Name = graphName
+                });
+
+            // Add a vertex collection
+
+            string clxToDelete = nameof(DeleteVertexCollectionAsync_ShouldSucceed);
+
+            PostVertexCollectionResponse createVertexResponse = await _client.PostVertexCollectionAsync(
+                graphName,
+                new PostVertexCollectionBody()
+                {
+                    Collection = clxToDelete
+                });
+
+            var response = await _client.DeleteVertexCollectionAsync(graphName, clxToDelete, new DeleteVertexCollectionQuery
+            {
+                DropCollection = false
```

Comment:
Let's also add a test with `DropCollection = true` to make sure the `DeleteVertexCollectionQuery` is transferred correctly.